src/components/Search.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+class Search extends React.Component {
+
+    constructor() {
+        super();
+
+        this.state= {
```

Comment:
It looks like query is being stored both in `Search` and `App`. Rather than duplicating state it would be better to pass the query down as props from `App` into `Search`